litebird_sim/simulations.py

```diff
@@ -209,3 +233,66 @@ def flush(self):
             self.base_path / "report.html", "w", encoding="utf-8",
         ) as outf:
             outf.write(html_full_report)
+
+    def create_observations(
```

Comment:
I'm not clear about the case of use of this method. The idea is that the Observation instances will create themselves later?

Reply:
No, the method `create_observations` initializes a sequence of `Observation` objects (line 264). This is the method that will be called in virtually *any* pipeline written using our `litebird_sim` framework, following this logic:

```python
sim = lbs.Simulation()
detectors = [lbs.Detector(params) for params in …]
lbs.create_observations(detectors, …)
# Etc.
```

Reply:
What `_` would represent here? I'm not clear if this stage should allocate the TOD etc or it is just creating a container for the observation setup and metadata

Reply:
In this stage the time span of the whole simulation is split among detectors. The real TOD must be created by the scanning strategy simulator; the function is just creating the «containers», as you call it. It's at this stage that the code slices the timespan figures out which MPI process will process each of them.

Reply:
Ok, thanks. I was wondering whether every detector should store quantities that are shared by all of them, like the sampling frequency

Reply:
Mmm… At the moment the sampling frequency is the same among the detectors, but I would not bet it will stay as it is forever. But I agree, there are some types of information that would be worth to be shared by many detectors, like the quaternion representing the rotation of the spacecraft. This does not fit in the `Observation` class, so it should perhaps be part of the `Simulation` object.